### ISSUE TITLE
Dead dbh for PD trees

### ISSUE THREAD

**Issue by emacmonigle** [SCBI-ForestGEO/SCBImortality#89]
Hi @jess-shue, since the last update dead dbh is required for DS trees that were PD. According do the census we don't need this. It also requires crown position, lianas, and dead dbh for DC/PD trees, all which we don't need/can't be collected. @teixeirak if any of the things I described are indeed needed, please let me know and we can reflect it in the protocol.

It would be great to have this changed in FF, but I know you said previously that changing one thing can affect others, so keeping them is fine with me, but maybe it should be noted post-collection so this data can be removed from DS and DC trees since they're not accurate (ex: no way to collect dead crown position from a fallen tree!)

**Comment by jess-shue** [comment]
@emacmonigle Things can be changed, I just can't make the rules too complicated or we run into issues. I had issues with making rules that apply to the 2021 status and the 2022 status (If PD in 2021 and DS/DC 2022 don't require etc.). I can try again and see if we can make things work.

We had decided in issue #82 to make the liana field required. We also decided in issue #81 to require crown position. But, I understand that for DC/PD trees this doesn't make sense - again, let me try a new rule!

Do you have data 'In Progress'? I won't publish changes if you do.

**Comment by emacmonigle** [comment]
@jess-shue I have an in progress quad from yesterday, and am about to head into the field. I won't have anything in progress by the end of today, since its the end of my field week. Would updating past 6pm today/tomorrow morning work for you?

**Comment by jess-shue** [comment]
@emacmonigle That works! Thank you for bringing up these issues - there is a lot of trial-and-error with this unfortunately right now because of adding these extra rules. I'll reply here when things are updated.

**Comment by jess-shue** [comment]
OK, @emacmonigle, looking at the protocol again and testing out rules, I THINK it is better now. Again, the more complicated we try to make these rules the more restrictions or problems we make for entering the data so I've gone simpler. It is now on Version 22.

Some requirements have been removed, but the data fields should show as needed - dead DBH and dead crown position for example. The requirement for a dead DBH if a tree was Alive in 2021 but is now DS/DC is no longer set, but the field does show up in yellow.

I've left the liana as a 'required' field. @teixeirak We debated the liana requirement in issue #82, and in the protocol '0 = lianas absent from tree crown'. Having it be required can be annoying, so we can turn it off, but you'll end up with blanks in that column if Erin forgets to choose something. We could assume that those blanks = 0, but that could be wrong....thoughts?
